pages/database-management/configuration.mdx

```diff
@@ -305,15 +305,17 @@ Memgraph contains settings that can be modified during runtime using a Cypher qu
-|----------------------------|-----------------------------------------------------------------------------------------------|-------------------------|
-| organization.name          | Name of the organization using the instance of Memgraph (used for verifying the license key). | yes                     |
-| enterprise.license         | License key for Memgraph Enterprise.                                                          | yes                     |
-| server.name                | Bolt server name.                                                                             | yes                     |
-| query.timeout              | Maximum allowed query execution time. Value of 0 means no limit.                              | yes                     |
-| log.level                  | Minimum log level. Allowed values: TRACE, DEBUG, INFO, WARNING, ERROR, CRITICAL.              | no                      |
-| log.to_stderr              | Log messages go to `stderr` in addition to `logfiles`.                                        | no                      |
-| cartesian-product-enabled  | Enforces cartesian product operator during query matching.                                    | no                      |
+| Setting name               | Description                                                                                                 | Persistent between runs |
+|----------------------------|-------------------------------------------------------------------------------------------------------------|-------------------------|
+| organization.name          | Name of the organization using the instance of Memgraph (used for verifying the license key).               | yes                     |
+| enterprise.license         | License key for Memgraph Enterprise.                                                                        | yes                     |
+| server.name                | Bolt server name.                                                                                           | yes                     |
+| query.timeout              | Maximum allowed query execution time. Value of 0 means no limit.                                            | yes                     |
+| log.level                  | Minimum log level. Allowed values: TRACE, DEBUG, INFO, WARNING, ERROR, CRITICAL.                            | no                      |
+| log.to_stderr              | Log messages go to `stderr` in addition to `logfiles`.                                                      | no                      |
+| cartesian-product-enabled  | Enforces cartesian product operator during query matching.                                                  | no                      |
+| hops_limit_partial_results | If set to `true`, partial results are returned after the hops limit is reached. The default value is `true`.| yes                     |
```

Comment:
What happens when this is `false`?

Reply:
Partial results are not returned, instead exception is thrown. Should we add that also?